controllers/feeds/getFeedsV2.js

```diff
@@ -110,11 +111,20 @@ module.exports = async (req, res) => {
 
   try {
     const {token} = req;
-    const myAnonymousUser = await UsersFunction.findAnonymousUserId(User, req.userId, {raw: true});
-    // START get excluded post parameter
-    const listBlockUser = await getListBlockUser(req.userId);
-    const listBlockDomain = await getBlockDomain(req.userId);
-    const listPostAnonymousAuthor = await getListBlockPostAnonymousAuthor(req.userId);
+    const [
+      myAnonymousUser,
+      // START get excluded post parameter
+      listBlockUser,
+      listBlockDomain,
+      listPostAnonymousAuthor,
+      isBlurredPost
+    ] = await Promise.all([
+      UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
+      getListBlockUser(req.userId),
+      getBlockDomain(req.userId),
+      getListBlockPostAnonymousAuthor(req.userId),
+      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId)
+    ]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The Promise.all() function is used here to execute multiple promises concurrently, which can improve performance by not waiting for each promise to resolve sequentially. However, it's important to note that if any promise in the array fails, Promise.all() will reject with the reason of the first promise that rejected. So, make sure all promises are reliable or add error handling for each promise.
```diff
-    const myAnonymousUser = await UsersFunction.findAnonymousUserId(User, req.userId, {raw: true});
-    const listBlockUser = await getListBlockUser(req.userId);
-    const listBlockDomain = await getBlockDomain(req.userId);
-    const listPostAnonymousAuthor = await getListBlockPostAnonymousAuthor(req.userId);
+    const [
+      myAnonymousUser,
+      listBlockUser,
+      listBlockDomain,
+      listPostAnonymousAuthor,
+      isBlurredPost
+    ] = await Promise.all([
+      UsersFunction.findAnonymousUserId(User, req.userId, {raw: true}),
+      getListBlockUser(req.userId),
+      getBlockDomain(req.userId),
+      getListBlockPostAnonymousAuthor(req.userId),
+      UserFollowUserFunction.checkIsBlurredPost(UserFollowUser, req.userId)
+    ]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->